prometheus/metrics.go

```diff
@@ -21,13 +21,15 @@ var (
 // MetricsQuery is a common struct for ServiceMetricsQuery and NamespaceMetricsQuery
```

Comment:
this comment is no longer valid - those ServiceMetricsQuery and NamespaceMetricsQuery no longer exist